install_template/templates/platformBase/ubuntu-20.04.njk

```diff
@@ -3,13 +3,13 @@
 {% block prerequisites %}
```

Comment:
Let's go with platformBase/ubuntu.njk or platformBase/ubuntu-18-plus.njk or similar and then have both ubuntu templates derive from it - 20 will likely not be the last version of Ubuntu we support and deriving 18 from 20 is nonintuitive.

Reply:
Just to be clear, you're suggesting I create a file called platformBase/ubuntu.njk, and use that as the basis for different versions ofUbuntu, such as ubuntu-18.04.njk and ubuntu-20.04.njk. Correct? 

Should this approach be used across the board? Currently, for some SLES products, SLES 15 is based on SLES 12. That seems reasonable to me. I agree that basing Ubuntu 18 off of Ubuntu 20 is non-intuitive.